fastlane/Fastfile

```diff
@@ -243,13 +243,13 @@ platform :ios do
   lane :lumenBeta do
 
     # Configuring Sentry DSN
-    sh("cd .. && plutil -replace SentryDSN -string $SentryDSN Client/Info.plist")
+    sh("sed -i '' 's|sentryDSN:\ String?\ =\ nil|sentryDSN:\ String?\ =\ \"'$SentryDSN'\"|' ../Cliqz/Resources/APIKeys.swift")
 
     # Configuring Revenuecat APIs
-    sh("cd .. && plutil -replace RevenuecatAPIKey -string $RevenuecatAPIKey Client/Info.plist")
+    sh("sed -i '' 's|lumenAPI:\ String?\ =\ nil|lumenAPI:\ String?\ =\ \"'$lumenAPIKey'\"|' ../Cliqz/Resources/APIKeys.swift")
```

Comment:
LumenAPI and revenueCatAPI keys order is wrong. Comment says RevCat but it replaces lumenAPI key and vice a versa.